package.json

```
@@ -221,7 +221,7 @@
         "haskell.plugin.tactic.config.max_use_ctor_actions": {
           "title": "Max number of constructors",
           "scope": "resource",
-          "type": "boolean",
+          "type": "number",
```

```suggestion
          "type": "integer",
```

It is more precise and should work, no?

---

Good point! I've added it in https://github.com/haskell/haskell-language-server/pull/1576/commits/0bda5272a84f376e46e2ffc0d301f4051365d578